### ISSUE TITLE
Croatian keyboard Č capitalization is backwards

### ISSUE THREAD

**Issue by toddlprice** [unfoldingWord-dev/ts-android#1668]
Make sure these boxes are checked before submitting your issue -- thank you!

- [x] Check that you are using the [latest version](https://github.com/unfoldingWord-dev/ts-android/releases/latest) of the app.
- [x] Check that a similar issue has not already been opened.

For tS running on Android (Lenovo TAB 2 A series), when using the Croatian keyboard, the character Č shows up capitalized when caps lock is _not_ on, but small (č) when it caps lock is on. This has happened on two tablets I use for Roma translators using the Croatian keyboard. All other letters work fine.

**Comment by PhotoNomad0** [comment]
@toddlprice - are they using translationKeyboard?  This seems like more of an issue with the keyboard program than ts-android.  I cannot find the Croatian keyboard to reproduce.  

My Lenovo does not have Croatian as a source language.  Also I could not find the Croatian keyboard for translationKeyboard.  @cckozie

**Comment by toddlprice** [comment]
No, they are not using translationKeyboard. There is not enough screen real estate for that on their tablets. I am using the Croatian keyboard accessible from Android. I used Setttings>Language Input. I have their tablets set for Croatian as the OS language. The weird thing is that it works fine on all other letters, except Č.

**Comment by PhotoNomad0** [comment]
Suspecting an issue with the external keyboard or a bug in android OS because we just add the characters Android gives us.  This would be very difficult to fix in the app, because we would have to add a check for each character coming in.  And we can't just reverse the cases, because the on screen keyboards do not show that problem.  

@neutrinog - joel do have any ideas here?
